app/views/quick_search/search/index.html.erb

```diff
@@ -44,7 +45,7 @@
   </div>
 </div>
 
-<% if Settings.LIBGUIDES.ENABLED %>
+<% if Settings.ENABLED_SEARCHERS.include?('lib_guides') %>
```

Comment:
If the change to how we feature flag "searchers" seems reasonable, I can also follow this to the logical conclusion of checking each block before to see if it is an enabled searcher (and then prob. do something Yewno specific since it's not really a "searcher").  The page won't flow nicely because these are all in rows (but maybe we can think about flexboxing these layouts and let them flow naturally).